admin/app/dfp/CustomTargetingAgent.scala

```diff
@@ -0,0 +1,69 @@
+package dfp
+
+import com.google.api.ads.dfp.axis.utils.v201705.StatementBuilder
+import com.google.api.ads.dfp.axis.v201705.{CustomTargetingKey, CustomTargetingValue}
+import common.Logging
+import common.dfp.{GuCustomTargeting, GuCustomTargetingValue}
+
+import scala.util.Try
+
+object CustomTargetingAgent extends DataAgent[Long, GuCustomTargeting] with Logging {
+
+  def loadFreshData(): Try[Map[Long, GuCustomTargeting]] = Try {
+    val maybeData = for (session <- SessionWrapper()) yield {
```

Comment:
You can generally omit the empty set of brackets like `SessionWrapper()` unless they are there to signify a side effect.